### ISSUE TITLE
Opening the fugitive git commit buffer causes the cursor to jump to minimap

### ISSUE THREAD

**Issue by akinsho** [wfxr/minimap.vim#43]
Hi, thanks for all your hard work on this plugin 👍🏿 

When trying to commit in vim using fugitive  hitting the `cc` shortcut to commit in the fugitive status buffer causes the cursor to jump to the `minimap` buffer (specifically, nowhere else, no matter how many buffers are open).

 I've never seen this behaviour and it only happens when this plugin is installed. I've used fugitive for years without this behaviour. When I remove this plugin nothing like this happens but with it installed this happens.

## Check list

- [x] I have read through the [README](https://github.com/wfxr/minimap.vim/blob/master/README.md) (especially F.A.Q section)
- [x] I have searched through the existing issues

## Environment info

- OS
    - [x] Linux
    - [x] Mac OS X
    - [ ] Windows
    - [ ] Others:

## Version info
```
NVIM v0.5.0-752-g759a05407
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversio
n -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGP
ACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/build/nvim/parts/nvim/build/build/config -I/build/nvim/parts/nvim/build/src -I/build/nvim/parts/nvim/build/.dep
s/usr/include -I/usr/include -I/build/nvim/parts/nvim/build/build/src/nvim/auto -I/build/nvim/parts/nvim/build/build/include
Compiled by root@lgw01-amd64-015

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

## Problem and steps to reproduce
Minimal vimrc
```
call plug#begin(stdpath('data') . '/plugged')
Plug 'wfxr/minimap.vim', {'do': ':!cargo install --locked code-minimap'}
Plug 'tpope/vim-fugitive'
call plug#end()
```

1. Open vim with the minimal vimrc, you need both `fugitive` and `minimap` installed. I'm not sure if this happens because of a direct interaction or if another circumstance would cause this
2. run `Gstatus` (you'll need to be in a git repo with a staged file)
3. Inside the `Gstatus` buffer, press `cc` this will open the commit window, but also the cursor will now be placed in the minimap buffer.

**Comment by wfxr** [comment]
@akinsho Thanks for your feedback. I confirmed this problem exists. It seems both `fugitive` and `minimap` tried to switch cursor between windows, which may caused this problem.

I think the easist way to solve the problem is to add a block list to disable specific file types.

**Comment by akinsho** [comment]
@wfxr that sounds good, tbh that was going to be a follow up/separate issue I was going to open tbh since there are a few buffers where I don't want/need a minimap, fugitive ones being a good example or for startup buffers like https://github.com/mhinz/vim-startify

**Comment by alerque** [comment]
This appears to have been addressed in 63a8a86.

**Comment by akinsho** [comment]
Yep I'll close this out now
